scripts/integration/default_set.yaml

```diff
@@ -1,12 +1,16 @@
+config-file: vast.conf
+
 fixtures:
   BiTopologyTester:
     enter: | # python
       node0 = Server(self.cmd,
-                     ['-e', ':{}'.format(VAST_PORT),
+                     [f'--config-file={self.config_file}',
+                      '-e', ':{}'.format(VAST_PORT),
```

Comment:
Probably an oversight: format string interpolation and `.format` calls in the same statement.